parser/cue/cue.go

```diff
@@ -16,7 +16,7 @@ func (c *Parser) Unmarshal(p []byte, v interface{}) error {
 	if err != nil {
 		return fmt.Errorf("error occured when formatting cue: %v", err)
 	}
-	instance, err := r.Parse("name", out)
+	instance, err := r.Compile("name", out)
```

Comment:
is this change relevant to this pull request?

Reply:
Not a part of the initial intent, but without it, golangci-lint fails.

Reply:
Since `Parse` func will be deprecated as the [docs](https://godoc.org/cuelang.org/go/cue) said, there is no problem to use `Compile`